.github/workflows/check.yaml

```diff
@@ -39,16 +39,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
           python-version: "3.7"
       - name: Run image
-        uses: abatilo/actions-poetry@v2.0.0
+        uses: abatilo/actions-poetry@v2
```

Comment:
this one as well? maybe the version can be set to a variable?

Reply:
I'll update it, but I haven't seen that pattern used anywhere else so I don't really see any value in the added comploxity.

Reply:
maybe its fine as is then, i just remember an action i was using once broke something but maybe thats not common enough to justify pinning them. up to u

Reply:
I think it's good to pin them, but using a variable seems like unknown complexity